src/components/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -30,17 +32,27 @@ export interface AmbassadorCardProps {
   ambassador: AmbassadorType;
   onClose?: () => void;
   className?: string;
+  disableCardEffects?: boolean;
 }
 
 export default function AmbassadorCard(props: AmbassadorCardProps) {
-  const { ambassadorKey, ambassador, onClose, className } = props;
+  const { ambassadorKey, ambassador, onClose, className, disableCardEffects } =
+    props;
   const images = getAmbassadorImages(ambassadorKey);
   const mod =
     window?.Twitch?.ext?.viewer?.role === "broadcaster" ||
     window?.Twitch?.ext?.viewer?.role === "moderator";
+  const glareOpacity = disableCardEffects ? 0.0 : 0.5;
 
   return (
-    <div
+    <Tilt
+      tiltEnable={!disableCardEffects}
+      glareEnable={!disableCardEffects}
+      glareMaxOpacity={glareOpacity}
+      glareBorderRadius="1rem"
+      glarePosition="bottom"
+      scale={1.0}
+      perspective={2500}
```

Comment:
```suggestion
      perspective={2500}
      tiltReverse
```
Can we enable `tiltReverse`? It feels more natural to me?